github/util_v4.go

```diff
@@ -37,6 +37,14 @@ func githubv4IDSlice(ss []string) []githubv4.ID {
 	return vGh4
 }
 
+func githubv4IDSliceEmpty(ss []string) []githubv4.ID {
```

Comment:
Why is "Empty" in the name here? It looks like the function is a translation between a string slice and a githubv4.ID slice, and I'm not quite understanding how Empty fits in.

Reply:
If I remember it correctly, I had to add this here so we can set the list of `pull_request_bypassers` to **empty list**: 

```terraform
resource "github_branch_protection" "test" {
  ...
  required_pull_request_reviews {
    pull_request_bypassers = []
  }
}
```

Without this we cannot remove all bypassers if any were set before.

I didn't want to make changes to the `githubv4IDSlice` for the case some other logic would break so I added this new function. If you feel the function should be called differently, plese suggest a new name and I will amend it.